server/src/graphql/mutations/createApp.ts

```diff
@@ -14,6 +14,14 @@ export const createApp: MutationResolvers['createApp'] = async (
     throw new Error('Unauthorized');
   }
 
+  const apps = await prisma.app.findMany();
```

Comment:
would be better to do a `findMany` with the name directly and check the item 0 in the array if it's set

Reply:
This step should be done after the yup validation :)